packages/anon-aadhaar-react/src/prove.ts

```diff
@@ -22,17 +22,18 @@ import {
  * @see {@link https://github.com/privacy-scaling-explorations/anon-aadhaar}
  *   For more information about the underlying circuit and proving system.
  */
-export const proveWithWebProver = async (
+export const proveAndVerify = async (
```

Comment:
I think name proveAndVerify here is confused. Maybe we should rename it to createPCD for initAndProve. 
WDTY?